pwa/app/routes/_index.tsx

```diff
@@ -29,28 +54,26 @@ export const meta: MetaFunction = () => {
 };
 
 export default function Index() {
-  const { status } = useLoaderData<typeof loader>();
-
-  // Commit hash is string-replaced, so we need to ignore eslint and typescript errors.
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-expect-error
-  const commitHash = __COMMIT_HASH__ as string;
```

Comment:
Can we keep this here until we have a chance to throw it into the footer or something? good for debugging...